docs/usecases/running_tests_on_cluster_not_deployed_by_ocsci.rst

```diff
@@ -0,0 +1,218 @@
+possible, but because ocs-ci was not maintained with this use case in
+mind, it requires few additional steps as explained below. Without
+paying attention to these details, test run could fail or report invalid
+results (both false negative or false positive results are possible).
+
+.. note::
+
+    When the following issues are fixed, most of the *additional steps*
+    explained here won’t be necessary:
+
+    - `#2347`_ platform detection/validation
+    - `#2523`_ remove platform specific values from default_config.yaml file
+    - `#3042`_ allow test execution on already installed IPI cluster without
+      specifying platform details
+
+.. _`#2347`: https://github.com/red-hat-storage/ocs-ci/issues/2347
+.. _`#2523`: https://github.com/red-hat-storage/ocs-ci/issues/2523
+.. _`#3042`: https://github.com/red-hat-storage/ocs-ci/issues/3042
+
+Prerequisites
+-------------
+
+We assume that you have:
+
+- Admin access to OCP cluster, where OCS has been already installed.
+- Admin access to cloud platform account where OCP cluster is running, if you
+  plan to execute platform specific or disruption test cases on cloud platform
+  already supported by ocs-ci.
+- Cluster directory of this OCP/OCS cluster on your machine, with all
+  files as produced by ``openshift-install`` tool.
+- Local copy of ocs-ci, installed in virtualenv on your machine, see
+  *Installing* section in :doc:`/getting_started` guide for details.
+- `Red Hat bugzilla`_ account.
+
+.. _`Red Hat bugzilla`: https://bugzilla.redhat.com/
+
+Preparation of test execution command
+-------------------------------------
+
+On your machine, go to ocs-ci directory and activate it’s virtual env.
+Check that ``oc`` tool is available (if not, update ``PATH`` environment
+variable or you can create symlink for it in ``bin`` directory where you have
+ocs-ci installed). You don’t need to set ``KUBECONFIG`` environment variable.
+
+Tests are executed via ``run-ci`` command (a wrapper of `pytest cli tool`_).
+But since ocs-ci was not used to deploy the cluster (and issues like `#2347`_),
+you have to specify the following information via command line options of
+``run-ci`` command.
+
+================================================================ ======================
+Information                                                      ``run-ci`` option
+================================================================ ======================
+Path to openshift cluster dir (from ``openshift-install`` tool)  ``--cluster-path=PATH``
+Path to ocs-ci config file with platform and deployment info     ``--ocsci-conf=PATH``
+Openshift cluster name (chosen during OCP installation)          ``--cluster-name=NAME``
+OCP version                                                      ``--ocp-version=VERSION``
+OCS version (check OCS operator)                                 ``--ocs-version=VERSION``
+Is OCS using already released builds? Use option if yes.         ``--live-deploy``
```

Comment:
is `--deploy` also required? and if it's deployment, should we add `-m deployment`?

Reply:
The text is discussing test execution only, neither deployment nor teardown is involved.

But since we assume that the cluster wasn't deployed via ocs-ci, we need to provide cluster details to `run-ci` to that it runs the tests correctly, as noted there. This includes `--live-deploy`, which will tell `run-ci` that the cluster is using stable builds (ocs-ci needs to know because of labels on catalog source and other details).